### PR TITLE
Improve to_bitmap parse int performance

### PR DESCRIPTION
**Pull request by kangkaisen**
`std::stoul` performance is bad， we shouldn't use it.

### CODE REVIEW

**Comment by imay** [comment]
> `std::stoul` performance is bad， we shouldn't use it.

@kangkaisen 
Is there any benchmark for `std::stoul`, better to create an issue to trace this problem. It will also be useful for others

**Comment by kangkaisen** [comment]
For https://github.com/apache/incubator-doris/issues/2224

File: be/test/util/string_parser_test.cpp [hunk 1]
```diff
@@ -209,13 +224,59 @@ TEST(StringToInt, Limit) {
...
+    test_unsigned_int_value<uint16_t>("12345", 12345, StringParser::PARSE_SUCCESS);
+    test_unsigned_int_value<uint32_t>("12345678", 12345678, StringParser::PARSE_SUCCESS);
+    test_unsigned_int_value<uint64_t>("12345678901234", 12345678901234, StringParser::PARSE_SUCCESS);
+
+    test_unsigned_int_value<uint8_t>("-10", 0, StringParser::PARSE_FAILURE);
+    test_unsigned_int_value<uint16_t>("-10", 0, StringParser::PARSE_FAILURE);
+    test_unsigned_int_value<uint32_t>("-10", 0, StringParser::PARSE_FAILURE);
+    test_unsigned_int_value<uint64_t>("-10", 0, StringParser::PARSE_FAILURE);
+
+    test_unsigned_int_value<uint8_t>("+1", 0, StringParser::PARSE_FAILURE);
```
Comment (imay) [inline be/test/util/string_parser_test.cpp]: Emm, number with plus sign is also a valid unsigned integer, is it appropriate to fail when parsing it？

Comment (kangkaisen) [reply be/test/util/string_parser_test.cpp]: 1.  unsigned integer means don't have a "+" or "-" sign
2. In Arrow, unsigned integer parser don't accept "+" or "-" sign
3. If we think "+" is  valid, the "-" should also be valid.

Comment (imay) [reply be/test/util/string_parser_test.cpp]: OK

**Review by imay — APPROVED** [review]
LGTM

**Pull request merged**